src/config/index.ts

```diff
@@ -141,6 +141,8 @@ export const UPDATE_MANAGE_TRUSTS_REVIEW_LEGAL_ENTITIES_PAGE = 'update-manage-tr
 export const UPDATE_MANAGE_TRUSTS_TELL_US_ABOUT_THE_LEGAL_ENTITY_PAGE = 'update-manage-trusts-tell-us-about-the-legal-entity';
 export const UPDATE_MANAGE_TRUSTS_INDIVIDUALS_OR_ENTITIES_INVOLVED_PAGE = 'update-manage-trusts-individuals-or-entities-involved';
 export const UPDATE_STATEMENT_VALIDATION_ERRORS_PAGE = "update-statement-validation-errors";
+// REMOVE TEMPLATES
+export const SOLD_ALL_LAND_FILTER_PAGE = "sold-all-land-filter";
```

Comment:
How about we prefix the constant name with `REMOVE_` and value with `remove-`?

Reply:
Makes sense. It will be consistent with the 'update' naming convention